classes/03class/exercises/c03-docker06/gepifanio/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM alpine:3.12.0
+ENTRYPOINT echo Guilherme
```

Comment:
still not this.
You specify the name during container execution, the command and output should be something as:
```
$ docker run hello-name:v1 Guilherme
Hello Guilherme
```

Reply:
Oh ok, hehe got it now.. I will fix it :)